src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataUpdater.java

```diff
@@ -104,8 +104,7 @@ protected boolean somethingChangedInBloggerData(Blog blog, BloggerEntry bloggerE
 
     private boolean urlFromRssIsValidAndDifferentThanExistingOne(Blog blog,
```

Comment:
I think that this method should be refactored - somethingChangedInBloggerData().
Can be case, that method: urlFromRssIsValidAndDifferentThanExistingOne() returns false, but one of the conditions above in somethingChangedInBloggerData() returns true and BloggerEntry will be added.